contracts/modules/TransferManager/VolumeRestrictionTMStorage.sol

```diff
@@ -25,6 +27,15 @@ contract VolumeRestrictionTMStorage {
         uint256 dailyLastTradedDayTime;
     }
 
+    struct RestrictedHolder {
+        // 1 represent true & 0 for false
+        uint8 seen;
+        // Type of period will be enum index of TypeOfPeriod enum
+        uint8 typeOfPeriod;
+        // Index of the array where the holder address lives
+        uint64 index;
```

Comment:
We can use `uint128` instead of `uint64`

Reply:
Yes, we could But 64 also work for us. For filling the remaining space we could use 128

Reply:
Let's just go with uint128 because we have space. No need to limit ourself where we don't need to :)